pom.xml

```diff
@@ -1083,6 +1407,26 @@
                             </execution>
                         </executions>
                         <dependencies>
+                            <dependency>
+                                <groupId>org.jboss.spec.javax.security.jacc</groupId>
+                                <artifactId>jboss-jacc-api_1.5_spec</artifactId>
+                                <version>${version.org.jboss.spec.org.jboss.spec.javax.security.jacc}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.jboss.logmanager</groupId>
+                                <artifactId>jboss-logmanager</artifactId>
+                                <version>${version.org.jboss.logmanager}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.wildfly</groupId>
+                                <artifactId>wildfly-client-all</artifactId>
```

Comment:
Not sure if this is the correct dependency, we should be able to perform this check just against the WildFly Elytron 1.7.0.Final release which is also a single jar.

Reply:
Without it it throws ERROR: Execution default of goal com.github.siom79.japicmp:japicmp-maven-plugin:0.13.0:cmp failed: Could not load 'org.wildfly.client.config.ConfigXMLParseException': org.wildfly.client.config.ConfigXMLParseException. Please make sure that all libraries have been added to the classpath (CLASSPATH=) or try the option '--ignore-missing-classes'.

Should I add ignore-missing-classes?

Reply:
Yes please try avoiding the missing classes if that is an option as in this test we only need to test the classes in our project.